packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -314,19 +396,20 @@ function FindUsername({ username, onConnect, onDone, onClose, onUsernameChange =
                             onKeyDown={onKeyDown}
                             inputProps={{ 'data-testid': 'username_input' }}></TextField>
                         <IconButton
-                            className={findUsernameClasses.button}
+                            className={isMobile ? classes.hide : findUsernameClasses.button}
```

Comment:
Don't need to define a special `hide` class, try to write the style by this way: [R103](https://github.com/DimensionDev/Maskbook/pull/1942/files#diff-b52ec66b9b9231ac15472a4a425dd04bacd3bb2becda880a75d78cae10dc8570R103)